guides/release/models/creating-updating-and-deleting-records.md

```diff
@@ -19,9 +19,9 @@ Making changes to Ember Data records is as simple as setting the attribute you
 want to change:
 
 ```javascript
-this.store.findRecord('person', 1).then(function(tyrion) {
+this.store.findRecord('post', 1).then(function(post) {
   // ...after the record has loaded
-  tyrion.firstName = 'Yollo';
+  post.title = 'UPDATED: Rails is Omakase';
```

Comment:
In the subsection `Persisting Records`, the code shows,

```javascript
store.findRecord('post', 1).then(function(post) {
  post.title; // => "Rails is Omakase"

  post.title = 'A new post';

  post.save(); // => PATCH to '/posts/1'
});
```

For continuity, can we either use `A new post` in your line 24, or replace `A new post` with `UPDATED: Rails is Omakase`?

Reply:
Addressed in 39a216a8